tests/platform_helper/test_command_conduit.py

```diff
@@ -170,22 +172,64 @@ def test_create_addon_client_task(
     access,
     addon_type,
     addon_name,
-    mock_application,
 ):
     """Test that, given app, env and permissions, create_addon_client_task calls
     get_connection_secret_arn with the default secret name and subsequently
-    subprocess.call with the correct secret ARN."""
+    subprocess.call with the correct secret ARN and execution role."""
     from dbt_platform_helper.commands.conduit import create_addon_client_task
 
+    env = "development"
+    mock_application = Mock()
+    mock_application.name = "test-application"
+    mock_application.environments = {"development": Mock()}
+
     task_name = mock_task_name(addon_name)
```

Comment:
Have the line break after the arrange lines...
```suggestion
    task_name = mock_task_name(addon_name)
    
```